folium/utilities.py

```diff
@@ -1,18 +1,21 @@
 # -*- coding: utf-8 -*-
-'''
+"""
 Utilities
 -------
 
 Utility module for Folium helper functions.
 
-'''
+"""
 
 from __future__ import absolute_import
 from __future__ import print_function
 from __future__ import division
+
+import time
```

Comment:
It is used in [this line](https://github.com/ocefpaf/folium/blob/pep8/folium/utilities.py#L260).  I am unsure of the consequences of its absence until today, but I am slowly revising `utilities.py`.